70_Pushover.pm

```diff
@@ -1300,6 +1329,7 @@ sub Pushover_SetMessage2 ($$$$) {
             my $rAck      = "cbAck_" . $1;
             my $rAckAt    = "cbAckAt_" . $1;
             my $rAckBy    = "cbAckBy_" . $1;
+            my $rAckByDev = "cbAckByDev_" . $1;
```

Comment:
3) The actual device that acknowledge the message is being recorded now - this is especially useful when sending to groups